src/main/resources/schema.graphqls

```diff
@@ -62,7 +62,7 @@ type RunStatus {
 }
 
 type RunListResponse {
-    runs: [RunStatus]!
+    runs: [RunResponse]!
```

Comment:
just want to confirm, the `state` field in both `RunResponse` and `RunStatus` are equivalent eh? If so, this looks good to me!

On another note, `state` sounds like it should be an enum to me, slight scope creep, but would be a nice improvement if possible :)

Reply:
Yes, the state is the same in `RunResponse` and `RunStatus`.